packages/tools/src/utilities/contours/interpolation/createInterpolatedToolData.ts

```diff
@@ -0,0 +1,58 @@
+import { Types, utilities as csUtils } from '@cornerstonejs/core';
+import { InterpolationROIAnnotation } from '../../../types/ToolSpecificAnnotationTypes';
+
+/**
+ * Creates a new annotation instance given the tool data, based on the referenced tool
+ * data type.
+ * Note that this object takes ownership of the polyline and handlePoints data, that is,
+ * directly assigns them internally to the result.
+ *
+ * @param polyline - data for the polyline, owned hereafter by the annotation
+ * @param handlePoints - data for the edit handles, if any, owned hereafter by the annotation
+ * @param referencedToolData
+
+ */
+export default function createInterpolatedToolData(
```

Comment:
It should contain contour or something to shows it is not lengthToolData, you know what I mean?